client/src/components/Workflow/Invocation/Export/ExistingInvocationExportProgressCard.vue

```diff
@@ -0,0 +1,105 @@
+<script setup lang="ts">
+import { BCard } from "bootstrap-vue";
+import { computed, ref } from "vue";
+
+import type { TaskMonitor } from "@/composables/genericTaskMonitor";
+import { getStoredProgressData, type MonitoringRequest } from "@/composables/persistentProgressMonitor";
+
+import PersistentTaskProgressMonitorAlert from "@/components/Common/PersistentTaskProgressMonitorAlert.vue";
+
+const monitoringSource = "wizard";
+
+interface Props {
+    invocationId: string;
+    useStsMonitor: TaskMonitor;
+    useRemoteMonitor: TaskMonitor;
+    exportToRemoteTaskId?: string;
+    exportToRemoteTargetUri?: string;
+    exportToStsRequestId?: string;
+}
+
+const props = defineProps<Props>();
+
+const emit = defineEmits<{
+    (e: "onDismissSts"): void;
+    (e: "onDismissRemote"): void;
+}>();
+
+const hasExistingStsExportData = ref(false);
+const hasExistingRemoteExportData = ref(false);
+
+const exportToStsRequest = computed<MonitoringRequest>(() => ({
+    source: monitoringSource,
+    action: "export",
+    taskType: "short_term_storage",
+    object: {
+        id: props.invocationId,
+        type: "invocation",
+    },
+    description: `Exporting invocation ${props.invocationId} to Short Term Storage in preparation for download`,
+}));
+
+const exportToRemoteRequest = computed<MonitoringRequest>(() => ({
+    source: monitoringSource,
+    action: "export",
+    taskType: "task",
+    object: {
+        id: props.invocationId,
+        type: "invocation",
+    },
+    description: `Exporting invocation ${props.invocationId} to remote source`,
+    remoteUri: props.exportToRemoteTargetUri,
+}));
+
+function updateExistingExportProgress() {
+    const existingStsExportData = getStoredProgressData(exportToStsRequest.value);
+    hasExistingStsExportData.value = existingStsExportData !== null;
+
+    const existingRemoteExportData = getStoredProgressData(exportToRemoteRequest.value);
+    hasExistingRemoteExportData.value = existingRemoteExportData !== null;
+}
+
+function dismissStsExport() {
+    emit("onDismissSts");
+    hasExistingStsExportData.value = false;
+}
+
+function dismissRemoteExport() {
+    emit("onDismissRemote");
+    hasExistingRemoteExportData.value = false;
+}
+
+updateExistingExportProgress();
+
+defineExpose({
+    updateExistingExportProgress,
+});
+</script>
+
+<template>
+    <BCard v-show="hasExistingStsExportData || hasExistingRemoteExportData">
```

Comment:
Minor improvment:
```suggestion
    <BCard v-show="hasExistingStsExportData || hasExistingRemoteExportData" class="mb-2">
```
|Before|After|
|---|----|
|![Screenshot From 2024-09-17 15-13-18](https://github.com/user-attachments/assets/626482ad-b657-4927-920c-8adc2ae6f6c6)|![Screenshot From 2024-09-17 15-12-58](https://github.com/user-attachments/assets/45404021-f576-4d76-a302-e94780bf2666)|